packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -679,16 +679,16 @@ export const prepareCosmosOrchestrationAccountKit = (
            * @type {OfferHandler<
            *   Vow<void>,
            *   {
-           *     amount: AmountArg;
            *     destination: ChainAddress;
+           *     amount: AmountArg;
            *     opts: IBCMsgTransferOptions;
            *   }
            * >}
            */
-          const offerHandler = (seat, { amount, destination, opts }) => {
+          const offerHandler = (seat, { destination, amount, opts }) => {
```

Comment:
unmotivated. don't we keep property keys in alphabetical order?

Reply:
My brain wanted to see them destructured in the same order as the positional arguments a few lines dow. But reverted in favor of alphabetical